### ISSUE TITLE
Loss of precision on int64s in numpy >= 1.22

### ISSUE THREAD

**Issue by TedSinger** [alexhsamuel/fixfmt#40]
```python
import fixfmt
import numpy
fmt = fixfmt._ext.Number(19, -1, pad=' ', sign=' ' , nan='NaN', inf='inf', point='.', bad='#')
x = 1600000000000000127
print(fmt(x))
# 1600000000000000127
print(fmt(numpy.int64(x)))
# 1600000000000000000
```
numpy 1.22.0 and up all reveal this issue, but 1.21.6 does not.

**Comment by alexhsamuel** [comment]
Reproduced; thanks for reporting.  I'll take a look.
Happy to see someone using this repo.

**Comment by alexhsamuel** [comment]
Fixed in 0.13.3.

**Comment by TedSinger** [comment]
Thank you!
